### DISCUSSION TITLE
Delaware County and Childhood Hunger

### DISCUSSION THREAD

**Discussion by spiritedgoldfish** [katskillskat/katskills-org#4] [General]
Poverty is not a topic I am well versed in writing about. However, I am well versed in living it. I know now that many other people in this area are as well. 

Days of my free school lunch being the only meal I ate were much less than occasional in elementary.  It was inevitable that my family would run out of money by the end of the month. I was raised to never speak of being "less fortunate" because starving children in Africa existed. I had tried to single-handedly dissolve that issue, however, my mother didn't ever manage to turn in my UNICEF boxes that I had taken around the neighborhood on Halloween. 

Several times, the power would be turned off and my father would run the house off of a generator so he could watch TV and drink a 30 rack. Our pipes would freeze from no heat and we would have to haul water from the river to boil it on the stove so my mother could wash her hair for work. Discussions involving our finances were not kept to a home office or a kitchen table between parents in the night, it was a constant screaming match that took place anywhere, at any time. 

Classism showed me its teeth when I was relentlessly taunted for not being able to pack my lunch for a sixth-grade field trip to the city. Instead, I used an excuse and got back on the bus during the time dedicated to The Museum of Natural History. I ate what was left of the other kids' lunches. The shame was heavily offset by the fact that other parents gave their kids incredible food I'd never seen or heard of before. I think I tried to pretend that eating their food would provide some sort of osmosis that would make me better. 

Once I'd reached high school, I knew better than to enter the cafeteria at all. My lunch hour was usually spent in places like the nurse's office, the chorus room, or the band room. I would wander the halls and dodge administration until the period was over. I wasn't the only kid who did this. More of them hid in the locker rooms or played basketball in the gym. Students would hang out in their favorite teacher's room or go smoke in the bathrooms. All of this was to avoid the fact that we knew we didn't have money for lunch and sitting there with an empty tray in front of you was - at least socially - considered worse than being caught being somewhere you weren't supposed to be.

As I'm experiencing adulthood, I am finding that these experiences were not as singular as I'd once thought. While it all may look different for every individual, growing up here was not easy. I've watched my childhood friends become addicts and parolees simply by trying to survive the only way they were taught. To say our parents dropped the ball could be right at surface level, but we all know it is so much deeper than that. 

In an area of poor access to mental health services, under-educated, under-paid school staff, and overworked social workers, the "cracks" that other people like me fell through are turning into the size of valleys. I have realized that I also cannot single-handedly save Delaware County like I couldn't save the starving children in the commercials. A community can be the difference but it will only continue to stunt and damage the populace unless we choose to grow. I'm linking a few pages that contain information and demographics about our area. Data USA, specifically, has visualizations of this information for easier digestion. I encourage you to look and to keep looking. Turning your cheek and closing your eyes to the brutal realities of this area will only encourage it to collapse. 

We can prevent future generations from experiencing these traumas if we decide. We can choose to be better observers and listeners in our community. It will require action and advocacy. It will require compassion and persistence. It will require fat slices of humble pie served to those who are too comfortable to bother facing facts. I implore you to choose the place you're living and make a change. 

(https://datausa.io/profile/geo/delaware-county-ny/)
(https://www.census.gov/quickfacts/delawarecountynewyork)
(https://www.ny.gov/counties/delaware)
(https://www.delcony.us/)

**Comment by katskillskat** [comment]
My immediate reaction to reading this is sadness, then anger. I'm sad that this happened to you. Where were the support systems, elected officials, and local non-profits that should have prevented this type of experience for a child?

Moral equivalence and relative privation are awful tactics that enable the closing eyes to localized suffering. Well-nourished children in Delaware county could more readily extend a helping hand to children in Africa, rather than them both suffering. Put on our oxygen mask before attempting to help others, lest we both could die.

I don't have any insight into whether kids in Delaware county are going hungry this week. How would we even know? Maybe it's not actively tracked or it's swept under the rug. Maybe it's not a great political platform to say "our kids need food".

Let's just assume that it's happening, kids in school are hungry, for the sake of a thinking about solutions.

What can be done to get food to local kids in a way that doesn't ostracize? How can it be made sustainable? How can the solution remove as many people from 'owning the solution' -- so that the grown-ups don't continue to mess it up?

Just in case some local kids are hungry.

**Reply by spiritedgoldfish** [reply]
There is such a severe degree of shame entrenched in needing help. I wish there was a simple way to find where and when we all became so isolated and ashamed, perhaps it's the lead. While I think we're getting better, it's almost like acknowledging struggle at all can be perceived as a failure. Then again, the systems designed to help those who are struggling have ways of making it even worse. Someone reaching out during a mental health crisis may be involuntarily hospitalized, another heavy and messy disruption to a person already deeply struggling. A parent trying to get out of an abusive relationship could have their kids taken away in the extensive process, the only thing they're fighting for, removed. 
It is, in my opinion, best practice for me to assume that everyone is struggling more than I could even imagine. While egocentrism may lead one to believe that you are the only person who has ever felt the way you are feeling, suffering is universal and everyone will experience it. Even though my family needed help, we were raised to refuse it profusely due to the consequence of our parents' pride.
I believe projects like community gardens and community dinners would be great steps forward. Similarly, organizations akin to the one linked below would be life-changing and help greatly with sustainability. While I know there are some that are not, the majority of the food banks in my area are held in churches. This could be a barrier to people as well. 
I think that there is a way forward. I am, however, unsure of how the masses would fair emotionally in having the wool removed from their eyes and being forced to see the things they so willfully ignored. 
(https://www.phama.org/)
